cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -30,6 +30,141 @@ Cask Data Application Platform Release Notes
+
+- :cask-issue:`CDAP-16676` - Upgrade to use dataproc API v1beta2 to allow endpoint config
+
+- :cask-issue:`CDAP-16697` - Created Best Practices guide for Spark engine tuning
+
+- :cask-issue:`CDAP-16709` - Implemented performance improvements to joiner plugins to cap the required memory to around 4gb per executor instead of scaling up as the skewness of the join goes up. Joins can now also be performed in-memory if one side is small, and behavior on null keys can be chosen by the user.
+
+- :cask-issue:`CDAP-16815` - Added a metric records.updated in BigQuery sink. This will give a total of all the inserts, updates and upserts into the sink.
+
+- :cask-issue:`CDAP-16918` - Introduced a new REST API for getting all application details across all namespaces
+
+- :cask-issue:`CDAP-16929` - Added the ability to select a Custom Dataproc Image. The complete URI for the custom image should be specified.
+
+- :cask-issue:`CDAP-16975` - UI now adds the latest version of plugin, among the list of different versions of the plugin, when added from the sidepanel in pipeline studio. If the user has already chosen a specific version (older version) it defaults to that instead of the latest.
+
+- :cask-issue:`CDAP-16976` - UI resets the default version of plugins for specific user during upgrade. When users upgrade from 6.1.2 to 6.1.3 or later UI will reset the default version of plugin the user has already chosen. Post upgrade if the user uses the same plugin UI will choose the latest version of the same plugin.
+
+- :cask-issue:`CDAP-17000` - Changed default value of spark.network.timeout to 10 minutes to make pipeline execution more stable for shuffle heavy pipelines.
+
+
+Bug Fixes
+---------
+
+- :cask-issue:`CDAP-17003` - Removed redundant validations from BQ sink , this should reduce calls to BigQuery.getTable()
+
+- :cask-issue:`CDAP-16530` - Fixed joiner output schema generation to be deterministic, using the same ordering as they had in the input data.
+
+- :cask-issue:`CDAP-17016` - Verified on the change in the delta replicator. Whenever a pipeline was created and ran, the target table created in BQ side will take the primary key as clustering columns as expected. 
+
+- :cask-issue:`CDAP-15869` - Preview displays logical types as strings.
+
+- :cask-issue:`CDAP-16222` - Fixed the package references in the dynamic spark plugin to use io.cdap instead of co.cask.
+
+- :cask-issue:`CDAP-16340` - Fixed the joiner plugin to allow a nullable key on one side and a non-nullable key on the other.
+
+- :cask-issue:`CDAP-16367` - Fixed a bug where field lineage is incorrect when a source is directly connected to a sink.
+
+- :cask-issue:`CDAP-16487` - Fixed regex for empty filter in Wrangler UI
+
+- :cask-issue:`CDAP-16731` - Fixed a bug that the groupBy aggregator requires a different alias for the field name.
+
+- :cask-issue:`CDAP-16747` - Modified loading screen for preview tab. 
```

Comment:
if there is no more information about this, let's remove this

Reply:
removed